src/node/markdown/plugins/highlight.ts

```diff
@@ -116,16 +122,6 @@ export async function highlight(
     }
 
     const lineOptions = attrsToLines(attrs)
-    const cleanup = (str: string) => {
-      return str
-        .replace(
-          preRE,
-          (_, attributes) =>
-            `<pre ${vPre}${attributes.replace(' tabindex="0"', '')}>`
```

Comment:
Clean-up was doing two things: adding `v-pre` as well as removing styles. Split into two transformers to make it better scoped.